canonical-web-and-design/www.ubuntu.com
# New /download/iot/raspberry-pi-2-3 page with ubuntu server

## Done

- Server version of raspberry pi page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/download/iot/raspberry-pi-2-3](http://0.0.0.0:8001/download/iot/raspberry-pi-2-3)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1KgnSBa8mbR7qwlxxtqbE5a1wXh5GjtVums8TIB7IGVw/edit#)

## Issue / Card

Fixes #4955
